pylabrobot/plate_reading/molecular_devices_backend.py

```diff
@@ -0,0 +1,1154 @@
+
+  async def read_absorbance(
+    self,
+    plate: Plate,
+    wavelengths: List[Union[int, Tuple[int, bool]]],
+    read_type: ReadType = ReadType.ENDPOINT,
+    read_order: ReadOrder = ReadOrder.COLUMN,
+    calibrate: Calibrate = Calibrate.ONCE,
+    shake_settings: Optional[ShakeSettings] = None,
+    carriage_speed: CarriageSpeed = CarriageSpeed.NORMAL,
+    speed_read: bool = False,
+    path_check: bool = False,
+    kinetic_settings: Optional[KineticSettings] = None,
+    spectrum_settings: Optional[SpectrumSettings] = None,
+    cuvette: bool = False,
+    settling_time: int = 0,
+    is_settling_time_on: bool = False,
+  ) -> MolecularDevicesDataCollection:
+    settings = MolecularDevicesSettings(
+      plate=plate,
+      read_mode=ReadMode.ABS,
+      read_type=read_type,
+      read_order=read_order,
+      calibrate=calibrate,
+      shake_settings=shake_settings,
+      carriage_speed=carriage_speed,
+      speed_read=speed_read,
+      path_check=path_check,
+      kinetic_settings=kinetic_settings,
+      spectrum_settings=spectrum_settings,
+      wavelengths=wavelengths,
+      cuvette=cuvette,
+      settling_time=settling_time,
+      is_settling_time_on=is_settling_time_on,
+    )
+    commands: List[Union[Optional[str], Tuple[str, int]]] = [self._get_clear_command()]
+    if not cuvette:
+      # commands.extend()
+      commands.extend(
+        [
+          *self._get_plate_position_commands(settings),
+          self._get_strip_command(settings),
+          self._get_carriage_speed_command(settings),
+        ]
+      )
+    commands.extend(
+      [
+        *self._get_shake_commands(settings),
+        *self._get_wavelength_commands(settings),
+        self._get_calibrate_command(settings),
+        self._get_mode_command(settings),
+        self._get_order_command(settings),
+        self._get_speed_command(settings),
+        self._get_tag_command(settings),
+        *self._get_nvram_commands(settings),
+        self._get_readtype_command(settings),
+      ]
+    )
+
+    await self._send_commands(commands)
+    await self._read_now()
+    await self._wait_for_idle()
+    return await self._transfer_data(settings)
+
+  async def read_fluorescence(
+    self,
+    plate: Plate,
+    excitation_wavelengths: List[int],
+    emission_wavelengths: List[int],
+    cutoff_filters: List[int],
+    read_type: ReadType = ReadType.ENDPOINT,
+    read_order: ReadOrder = ReadOrder.COLUMN,
+    calibrate: Calibrate = Calibrate.ONCE,
+    shake_settings: Optional[ShakeSettings] = None,
+    carriage_speed: CarriageSpeed = CarriageSpeed.NORMAL,
+    read_from_bottom: bool = False,
+    pmt_gain: Union[PmtGain, int] = PmtGain.AUTO,
+    flashes_per_well: int = 10,
+    kinetic_settings: Optional[KineticSettings] = None,
+    spectrum_settings: Optional[SpectrumSettings] = None,
+    cuvette: bool = False,
+    settling_time: int = 0,
+    is_settling_time_on: bool = False,
+  ) -> MolecularDevicesDataCollection:
```

Comment:
I think `is_settling_time_on` can be inferred from `settling_time` not being 0? is there a reason to have both?